src/analytics/analytics.js

```diff
@@ -0,0 +1,155 @@
+/** @flow */
+import serializeError from 'serialize-error';
+import requestify from 'requestify';
+import hashObj from 'object-hash';
+import uniqid from 'uniqid';
+import yn from 'yn';
+import R from 'ramda';
+import omitBy from 'lodash.omitby';
+import isNil from 'lodash.isnil';
+import logger from '../logger/logger';
+import { setSync, getSync } from '../api/consumer/lib/global-config';
+import {
+  CFG_ANALYTICS_USERID_KEY,
+  CFG_ANALYTICS_REPORTING_KEY,
+  CFG_ANALYTICS_ERROR_REPORTS_KEY,
+  CFG_ANALYTICS_ANONYMOUS_KEY,
+  CFG_USER_EMAIL_KEY,
+  CFG_USER_NAME_KEY,
+  DEFAULT_BIT_ENV,
+  CFG_ANALYTICS_ENVIRONMENT_KEY,
+  ANALYTICS_URL
+} from '../constants';
+
+const LEVEL = {
+  DEBUG: 'debug',
+  INFO: 'info',
+  WARNING: 'warning',
+  ERROR: 'error',
+  FATAL: 'fatal'
+};
+
+class Breadcrumb {
+  message: string;
+  data: Object;
+
+  constructor(category: string, message: string, data: Object) {
+    this.category = category;
+    this.message = message;
+    this.data = data;
+  }
+}
+class Analytics {
+  static username: string;
+  static command: string;
+  static release: string;
+  static args: string[];
+  static flags: Object = {};
+  static success: boolean = true;
+  static nodeVersion: string;
+  static os: string;
+  static extra: ?Object = {};
+  static level: string;
+  static error: Error | string | Object;
+  static breadcrumbs: Array<Breadcrumb> = [];
+  static analytics_usage: boolean;
+  static error_usage: boolean;
+  static anonymous: boolean;
+  static environment: string;
+
+  static getID(): string {
+    const id = getSync(CFG_ANALYTICS_USERID_KEY);
+    const newId = uniqid();
+    if (id) return id;
+    setSync(CFG_ANALYTICS_USERID_KEY, newId);
+    return newId;
+  }
+
+  static _hashArgs(flags: Object) {
```

Comment:
you have 2 props one called args and one flags.
This function called hash args but getting flags.. change the name of something to be aligned